e2e/process-services-cloud/task-form-cloud-component.e2e.ts

```diff
@@ -104,7 +92,13 @@ describe('Task form cloud component', () => {
 
         formTaskId = formTasks.list.entries[0].entry.id;
 
-        await loginSSOPage.loginSSOIdentityService(testUser.email, testUser.password);
+        /* cspell: disable-next-line */
+        const assigneeProcessDefinition = await processDefinitionService.getProcessDefinitionByName(browser.params.resources.ACTIVITI_CLOUD_APPS.SIMPLE_APP.processes.calledprocess, simpleApp);
```

Comment:
if these preconditions are only related to the test C306872 they should be part of it. Otherwise, the afterAll will be confusing